examples/python/Misc/MultiDevice/external_sync_frame_sync_minimal.py

```
@@ -0,0 +1,179 @@
+            else:
+                time.sleep(0.001)
+
+    # Start pipelines
+    masterPipeline.start()
+    for _, slavePipeline in slavePipelines.items():
+        slavePipeline.start()
+
+    # Start threads
+    threads = {}
+    for deviceName, sockets in slaveQueues.items():
+        for socketName, camOutputQueue in sockets.items():
```

Ruff (0.15.12)

[warning] Loop control variable `camOutputQueue` not used within loop body

Rename unused `camOutputQueue` to `_camOutputQueue`

(B007)